Base.rte/Scripts/AirBlast.lua

```diff
@@ -13,10 +13,18 @@ function Update(self)
 				if strSumCheck < self.strength then
 					local massFactor = math.sqrt(1 + math.abs(mo.Mass));
 					local distFactor = 1 + dist.Magnitude * 0.1;
-					local forceVector =	dist:SetMagnitude((self.strength - strSumCheck) /distFactor);
-					mo.Vel = mo.Vel + forceVector /massFactor;
-					mo.AngularVel = mo.AngularVel - forceVector.X /(massFactor + math.abs(mo.AngularVel));
+					local forceVector =	dist:SetMagnitude((self.strength - strSumCheck)/distFactor);
```

Comment:
Looks like there's some odd whitespace here.

Reply:
Odd, I don't see anything on my end. Maybe I forgot something.

Reply:
It looks like there's a tab between the `=` and `dist` instead of a space?